project-tracker backend/src/app/controllers/milestone.ts

```diff
@@ -18,8 +18,8 @@ export const index = async (req: Request, res: Response) => {
 
 export const show = async (req: Request, res: Response) => {
   try {
-    const milestone: Milestone = await database(TableNames.milestones).select().where({ id: req.params.id }).first();
-    if (typeof milestone !== 'undefined') {
+    const milestone: Milestone = await database(TableNames.milestones).select().where({ id: req.params.id, deletedAt: null }).first();
```

Comment:
van külön ilyen, hogy whereNull: http://knexjs.org/#Builder-whereNull

Reply:
rastasi 22 hours ago: Tehát a nem törölteket where "deletedAt is null" -lal tudod lekérni.
de akkor átírom

Reply:
yep, sql-ben ugyanezt csinálja a whereNull is, csak átláthatóbb :)